app/controllers/tenants_controller.rb

```diff
@@ -1,5 +1,6 @@
 class TenantsController < ApplicationController
   def index
+    @tenants = Tenant.all.where(active: true).where(approved: true)
```

Comment:
This is for everyone, there is a user story where there is a tenants listing that changes based on admin changes, and this is setting it up.